src/components/Auth/ConfirmPassword/ConfirmPasswordComponent.tsx

```diff
@@ -38,51 +38,47 @@ const ConfirmPasswordComponent = () => {
 
       authenticate(data.token?.token)
       setRefreshToken(data.token?.refreshToken)
+      await setCookie('token', data.token?.token, { path: '/' })
 
       reset()
 
       redirectToPreviousRoute()
-    } catch (error: unknown) {
-      if (axios.isAxiosError(error)) {
-
-        const axiosError = error as AxiosError<ErrorResponse>
-
-        if (axiosError.response) {
-          setErrorMessage(`Server Error: ${axiosError.response.data.message}`)
-        }
-      } else {
-        setErrorMessage('An unknown error occurred')
-      }
+    } catch (error) {
+      handleError(error)
     } finally {
       setLoading(false)
     }
   }
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)} >
-      {errorMessage && (
-        <div className="mt-4 text-negative">
-          {errorMessage}
+    <>
+      <h1 className='text-[36px] text-primary font-medium mb-[16px]'>Confirm password</h1>
+      <p className='text-[18px] text-primary font-medium mb-[40px]'>An email has been sent to your email with a code to confirm your registration. Please check your e-mail box.</p>
```

Comment:
nit: replace with `Enter code that was sent to your email to confirm registration.`

Reply:
fix